package.json

```diff
@@ -10,16 +10,18 @@
     "@types/node": "^16.7.13",
     "@types/react": "^17.0.40",
     "@types/react-dom": "^17.0.13",
+    "classnames": "^2.3.1",
+    "normalize.css": "^8.0.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-scripts": "5.0.0",
     "typescript": "^4.6.2",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "react-app-rewired start",
```

Comment:
fyi: This project is "lightly maintained" - `craco` is probably a better option going forward

Reply:
the reason of choosing app-rewired was the package react-app-rewire-postcss for postcss out of the box without configuration.
I think, on current stage it's good for speed up